v1/topdown/eval.go

```diff
@@ -450,10 +529,6 @@ func (e *eval) evalStep(iter evalIterator) error {
 		return err
 	}
 
-	if !defined {
-		e.traceFail(expr)
-	}
-
 	return nil
```

Comment:
Since you're all over the place already, how about changing this
```
if err != nil {
  return err
}
return nil
```
to
```
return err
```
? 😃

Reply:
Will do!